### PR TITLE
Indicate breakpoint expiration using the refers_to field

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

File: test/test-debuglet.js [hunk 1]
```diff
@@ -658,8 +658,11 @@ describe('Debuglet', function() {
               .reply(200, {breakpoints: [bp]})
               .put(BPS_PATH + '/test',
                    function(body) {
-                     return body.breakpoint.status.description.format ===
-                            'The snapshot has expired';
+                     var status = body.breakpoint.status;
+                     return status.description.format ===
+                            'The snapshot has expired' &&
+                            status.refersTo ===
+                            'BREAKPOINT_AGE';
```
Comment (ofrobots) [inline test/test-debuglet.js]: nit: This probably fits on the previous line.

File: src/agent/debuglet.js [hunk 2]
```diff
@@ -650,7 +650,7 @@ Debuglet.prototype.scheduleBreakpointExpiry_ = function(breakpoint) {
         format: 'The snapshot has expired'
       },
       isError: true,
-      refersTo: 'unspecified'
+      refersTo: 'BREAKPOINT_AGE'
```
Comment (ofrobots) [inline src/agent/debuglet.js]: Please add this to status-message.js, and use a constant from there.

File: test/test-debuglet.js (hunk 1 above)
Comment (matthewloring) [inline test/test-debuglet.js]: Done.

File: src/agent/debuglet.js (hunk 2 above)
Comment (matthewloring) [inline src/agent/debuglet.js]: Done.

**Pull request merged**